### PR TITLE
RHDEVDOCS-4544-SBO-RN: Added a deprecation notice for OLM descriptors

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve [RHDEVDOCS-4544](https://issues.redhat.com/browse/RHDEVDOCS-4544): Release Notes, New Features and Known Issues for SBO 1.3

**Aligned team**: Dev Tools

**OCP version for cherry-picking**: enterprise-4.11 and later

**Preview pages**: https://51554--docspreview.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/sbo-release-notes.html#sbo-release-notes-1-3_servicebinding-release-notes

**SME Review completed**: @dperaza4dustbit 
**QE review completed**: @pmacik 
**Peer-review needed**: @jc-berger

### CODE REVIEW

**Comment by shipsing** [comment]
@dperaza4dustbit, Could you please help with SME review of this PR? Thanks

**Comment by shipsing** [comment]
@sadlerap @pmacik, Could you please help with SME review of this PR? Thanks

File: modules/sbo-release-notes-1-3.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
+= Release notes for {servicebinding-title} 1.3
+
+{servicebinding-title} 1.3 is now available on {product-title} 4.9, 4.10, and 4.11.
+
+[id="deprecation-notice-1-3_{context}"]
+== Deprecated functionality
+* In {servicebinding-title} 1.3, the Operator Lifecycle Manager (OLM) descriptor feature is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to OLM descriptors, you can use CRD annotations instead to declare binding data.
```
Comment (dperaza4dustbit) [inline modules/sbo-release-notes-1-3.adoc:12]: We are not deprecating, we are removing. Please use the text in release notes here https://issues.redhat.com/browse/APPSVC-1203. This feature is not being used today

**Review by dperaza4dustbit — CHANGES_REQUESTED** [review]

File: modules/sbo-release-notes-1-3.adoc (hunk 1 above)
Comment (shipsing) [reply modules/sbo-release-notes-1-3.adoc]: @dperaza4dustbit, we can not use the same RN text as mentioned in https://issues.redhat.com/browse/APPSVC-1203. I have modified the RN statement as per our supplemetary style guide. Please have a look.

File: modules/sbo-release-notes-1-3.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
+= Release notes for {servicebinding-title} 1.3
+
+{servicebinding-title} 1.3 is now available on {product-title} 4.9, 4.10, and 4.11.
+
+[id="deprecation-notice-1-3_{context}"]
+== Removed functionality
+* In {servicebinding-title} 1.3, the Operator Lifecycle Manager (OLM) descriptor feature has been removed to improve resource utilization. As an alternative to OLM descriptors, you can use CRD annotations instead to declare binding data.
```
Comment (dperaza4dustbit) [inline modules/sbo-release-notes-1-3.adoc:12]: lgtm

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Comment by pmacik** [comment]
LGTM

File: modules/sbo-release-notes-1-3.adoc (hunk 2 above)
Comment (jc-berger) [inline modules/sbo-release-notes-1-3.adoc:12]: Minimalism: Minor tweak but I think `instead` is redundant. 

`As an alternative to OLM descriptors, you can use CRD annotations to declare binding data.`

**Review by jc-berger — APPROVED** [review]
/lgtm

File: modules/sbo-release-notes-1-3.adoc (hunk 2 above)
Comment (shipsing) [reply modules/sbo-release-notes-1-3.adoc]: Done

File: modules/sbo-release-notes-1-3.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[id="sbo-release-notes-1-3_{context}"]
+// Module included in the following assembly:
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
```
Comment (bburt-rh) [inline modules/sbo-release-notes-1-3.adoc:5]: The `content-type` metadata needs to be placed before the anchor ID:

```
// Module included in the following assembly:
//
// * applications/connecting_applications_to_services/sbo-release-notes.adoc

:_content-type: REFERENCE
[id="sbo-release-notes-1-3_{context}"]
= Release notes for {servicebinding-title} 1.3
```

Comment (shipsing) [reply modules/sbo-release-notes-1-3.adoc:5]: done

**Comment by bburt-rh** [comment]
@shipsing has indicated that no CPs are required for this PR for the `enterprise-4.9` and `enterprise-4.10` branches, so the failed CPs for those branches are now moot.

**Pull request merged**